angular-clipboard/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for angular-clipboard v1.5
+// Project: https://github.com/omichelsen/angular-clipboard
+// Definitions by: Bradford Wagner <https://github.com/bradfordwagner/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'angular-clipboard' {
```

Comment:
could you strip `declare module 'angular-clipboard' {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred